pkgs/applications/editors/emacs/elisp-packages/manual-packages/cask/package.nix

```diff
@@ -1,30 +1,48 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, bash
-, emacs
-, python3
+{
+  lib,
+  ansi,
+  coreutils,
+  dash,
+  ecukes,
+  el-mock,
+  ert-async,
+  ert-runner,
+  f,
+  fetchFromGitHub,
+  git,
+  gitUpdater,
+  melpaBuild,
+  noflet,
+  package-build,
+  s,
+  servant,
+  shell-split-string,
 }:
 
-stdenv.mkDerivation (finalAttrs: {
-  pname = "cask";
+let
   version = "0.8.8";
+in
+melpaBuild {
+  pname = "cask";
+  inherit version;
 
   src = fetchFromGitHub {
+    pname = "cask-source";
+    inherit version;
```

Comment:
I am not sure about this change.  This is similar to https://github.com/NixOS/rfcs/pull/171, right?  I do not follow that RFC, but since it is closed, I guess we should not set `pname` and `version` here.

Reply:
I use this to track the various fetchers.
The experience of various `hash mismatch in /nix/store/<hash>-source.drv` is confusing and horrible.

Reply:
So it is only used for development?  What about not including it in the final expression?

BTW, there is only one hash for this package.  It would not be confusing if you build this package alone for development, right?

Reply:
Sometimes I test various more or less unrelated things at the same time.
It is easier than creating a lot of dedicated branches and swapping among them.
Further, `nixpkgs-review` makes it easier to test various modifications at the same time.
(Obviously, since GitHub forces a merge-branch style of development, I split the modifications into dedicated branches before uploading them.)

Reply:
I would say what you want is basically https://github.com/NixOS/rfcs/pull/171.  Pros and cons are discussed there.  There must be a reason to not accept that RFC, right?  In addition, I do not find any precedents in the repo in a quick search.  So I slightly against this.